tests/integration/ha/helpers.py

```diff
@@ -210,6 +210,40 @@ def cut_unit_network(machine_name: str) -> None:
     subprocess.check_call(cut_network_command.split())
 
 
+def cut_network_from_unit_without_ip_change(machine_name: str) -> None:
```

Comment:
**nitpick:** Maybe rename the function to `network_throttle()` and `network_release()`. Then we could have `network_cut()` and `network_restore` as normal. 

`*_from_unit_without_ip_change` seems overly verbose and not immediately obvious.